docs/ViewportSensor.md

```diff
@@ -59,3 +59,52 @@ how much to throttle document's `scroll` event.
 
 Is the same as `<ViewportSensor>`, but uses only `IntersectionObserver` to detect element's intersection
 with viewport.
+
+
+## `withViewport()` HOC
+
+HOC that merges `viewport` boolean prop into enhanced component's props. Uses `<ViewportSensor>` under-the-hood.
+
+```jsx
+import {withViewport} from 'libreact/lib/ViewportSensor';
+
+const MyComp = (props) =>
+  <pre style={{fontFamily: 'nomospace'}}>
```

Comment:
`nomospace` again. Find and replace please.

Reply:
fixed